src/style.css

```diff
@@ -0,0 +1,41 @@
+body {
+  margin: 0;
+}
+
+.header {
+  display: flex;
+  height: 60px;
+  justify-content: space-between;
+  align-items: center;
+  padding: 0 50px;
+}
+
+.nav-li {
+  display: inline;
```

Comment:
height of this elements has to be the same as the height of the header

Reply:
height property ignored here due to display: inline